grafana-plugin/src/state/rootBaseStore/index.ts

```diff
@@ -57,25 +50,7 @@ export class RootBaseStore {
   backendLicense = '';
 
   @observable
-  pluginIsInitialized = true;
-
-  @observable
-  correctProvisioningForInstallation = true;
-
-  @observable
-  correctRoleForInstallation = true;
-
-  @observable
-  signupAllowedForPlugin = true;
-
-  @observable
-  initializationError = '';
-
-  @observable
-  retrySync = false;
-
-  @observable
-  isUserAnonymous = false;
+  initializationError = null;
```

Comment:
this has been simplified. all of the above attributes that were removed have been collapsed into `initializationError`, see `grafana-plugin/src/plugin/PluginSetup/index.tsx` for more details